pandas/core/series.py

```diff
@@ -3617,7 +3618,7 @@ def map(self, arg, na_action=None):
 
         Parameters
         ----------
-        arg : function, dict, or Series
+        arg : function, dict, colletions.abc.Mapping subclass or Series
```

Comment:
dict is now superfluous

Reply:
Wasn't sure if I should remove it as I suppose many standard users are familiar with `dict` but unfamiliar with `Mapping`.
Removed it now.